journal/updated/07-chapter-6.adoc

```diff
@@ -606,16 +609,16 @@ in order that a few living stones may be supporters of a holy building.
 O, my dears, look to it, and remember what station you are in.
 In the love of the Gospel I beseech you to take heed how you walk,
 and that you so demean yourselves as to be ensamples to the flock of Christ,
-and feeders of his lambs--way-marks unto all, both within and without.
-We are watched, and we must watch also, even against every appearance of evil;
+and feeders of His lambs--way-marks unto all, both within and without.
+We are watched, and we must watch also, even against every appearance of evil,
 and not allow our own wills to prevail,
 but let our dispositions be brought into true subjection,
 that we may experience the truth of the lion and the lamb lying down together,
 and of the weaned child playing at the hole of the asp,
```

Comment:
lets change this so that it uses the NKJV terms. So instead of asp, put cobra. And instead of cockatrice, put viper.  Oh, and instead of "sucking child" put weaned child. I don't think anybody knows those words anymore.